src/Cardano/Shell/Constants/CLI.hs

```diff
@@ -116,18 +116,20 @@ configCoreCLIParser = PartialCore
           <> metavar "GENESIS-HASH"
           <> help "The genesis hash value."
            )
-    <*> lastStrOption
-           ( long "signing-key"
-          <> metavar "FILEPATH"
-          <> help "Path to the signing key."
-           )
-    <*> lastStrOption
-           ( long "delegation-certificate"
-          <> metavar "FILEPATH"
-          <> help "Path to the delegation certificate."
-           )
+    <*> ((Just <$>) <$> lastStrOption
```

Comment:
To be perfectly honest, I don't see the point of doing this.
Think about it, we already have a `Maybe`.

Reply:
@ksaric, consider that we need to detect if the end user supplied just one of the certificate and the key -- that's an error.

If we don't have the `Maybe`, we wouldn't be able to detect it.